resources/core/charts/jaeger/templates/virtualservice.yaml

```diff
@@ -8,14 +7,26 @@ spec:
   hosts:
   - jaeger.{{ .Values.global.domainName }}
   gateways:
-  - {{ .Values.global.istio.gateway.name }}
+  - core-{{ .Chart.Name }}-gateway
   http:
   - match:
-    - uri:
-        regex: /.*
     route:
     - destination:
+        host: core-{{ .Chart.Name }}
         port:
           number: {{ .Values.service.uiPort }}
-        host: core-{{ .Chart.Name }}
-{{- end }}
\ No newline at end of file
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: core-{{ .Chart.Name }}-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
```

Comment:
Why do you need http gateway?

Reply:
To be more restrictive in the reachability from outside. All the other ports exposed by Jaeger are available only from calls coming from services running inside of Kyma. We will define Istio authorization rules for them later (we have an issue for this) .

Reply:
Changed